packages/theme/src/theme/_buttons.scss

```diff
@@ -72,26 +89,18 @@
 	}
 
 	&-link {
+		@include svg-button-behavior;
 		font-size: $font-size-base;
 		font-weight: inherit;
-
-		&.btn-icon-only {
-			color: $gray;
-
-			&:focus,
-			&:hover {
-				color: $scooter;
-			}
-
-			&:active {
-				color: darken($scooter, 12%);
-			}
-		}
 	}
 
 	&-icon-only {
 		display: inline-block;
 
+		&.btn-link {
+			@include svg-button-behavior;
```

Comment:
It seems that this block is already covered by l.92

Reply:
It allows to override focus color from bootstrap.
But there is maybe a better way to do ?

![image](https://user-images.githubusercontent.com/35027619/55627890-8a23b500-57af-11e9-9e80-80129b1481e1.png)

Reply:
Not sure what you mean, but this is from our theme showroom.
I picked a button-icon-only element

* default state, the color is duplicated because of those lines
<img width="250" alt="Capture d’écran 2019-04-05 à 16 38 17" src="https://user-images.githubusercontent.com/10761073/55635639-7d0fc180-57c1-11e9-9538-a06d7696ac24.png">

* hover state, the color is also duplicated
<img width="228" alt="Capture d’écran 2019-04-05 à 16 38 54" src="https://user-images.githubusercontent.com/10761073/55635669-91ec5500-57c1-11e9-9121-be5bcee90089.png">

* same for the svg inside

What I wanted to tell is that L.101 override the same values L.92 introduces. So if you remove it, it produces the exact same values.
Am I missing something ?